.github/workflows/tests.yml

```diff
@@ -1,9 +1,11 @@
 name: Tests
 
 on:
+  pull_request: null
   push:
-    branches: [main]
-  pull_request:
+    branches:
+      - main
+  workflow_dispatch: null
```

Comment:
I don't get why enabling `workflow_dispatch` in this case, but either way it doesn't need this `null` value

Reply:
It's common that we have to enable "some triggers" manually, for testing.
But I understand that now it's not necessary.

Removing. :)